esdt-safe/src/from_sovereign/token_mapping.rs

```diff
@@ -12,54 +11,6 @@ pub struct EsdtTokenInfo<M: ManagedTypeApi> {
 pub trait TokenMappingModule:
     multiversx_sc_modules::default_issue_callbacks::DefaultIssueCallbacksModule
 {
-    #[payable("EGLD")]
-    #[endpoint(registerToken)]
-    fn register_token(
```

Comment:
Please revert this change.
In `esdt-safe` contract we still use `register_token`. Only in `enshrine-esdt` we don't use it anymore.

Reply:
reverted